src/olympia/users/models.py

```diff
@@ -180,6 +180,12 @@ def is_staff(self):
         from olympia.access import acl
         return acl.action_allowed_user(self, amo.permissions.ADMIN)
 
+    @property
+    def is_reviewer(self):
```

Comment:
This function makes me uneasy - it's likely to be misused at some point, even though the comments (correctly) point out it shouldn't be used for ACL checks.  As far as I can see it's only used in one place (in live code) anyway - just move `self.groups.filter(name__startswith='Reviewers: ').exists()` there?

Reply:
Agreed